### ISSUE TITLE
ValueSortedDict

### ISSUE THREAD

**Issue by gbtami** [grantjenks/python-sortedcontainers#150]
In https://github.com/gbtami/pychess-variants I use ValueSortedDict from your https://github.com/grantjenks/python-sortedcollections package. It worked Ok so far but today my CI unittests failed.
Seems your latest changes break sortedcollections 1.1.2
```
======================================================================

ERROR: test_lost_and_out (__main__.HighscoreTestCase)

----------------------------------------------------------------------

Traceback (most recent call last):

  File "/home/travis/virtualenv/python3.6.7/lib/python3.6/site-packages/aiohttp/test_utils.py", line 488, in new_func

    func(self, *inner_args, **inner_kwargs))

  File "/opt/python/3.6.7/lib/python3.6/asyncio/base_events.py", line 473, in run_until_complete

    return future.result()

  File "server/test.py", line 243, in test_lost_and_out

    await self.play_and_resign(game, self.wplayer)

  File "server/test.py", line 210, in play_and_resign

    await game.game_ended(player, "resign")

  File "/home/travis/build/gbtami/pychess-variants/server/game.py", line 680, in game_ended

    await self.save_game()

  File "/home/travis/build/gbtami/pychess-variants/server/game.py", line 331, in save_game

    await self.update_ratings()

  File "/home/travis/build/gbtami/pychess-variants/server/game.py", line 461, in update_ratings

    await self.set_highscore(self.variant, self.chess960, {self.wplayer.username: int(round(wr.mu, 0))})

  File "/home/travis/build/gbtami/pychess-variants/server/game.py", line 427, in set_highscore

    self.highscore[variant + ("960" if chess960 else "")].update(value)

  File "/home/travis/virtualenv/python3.6.7/lib/python3.6/site-packages/sortedcontainers/sorteddict.py", line 551, in update

    self._setitem(key, pairs[key])

  File "/home/travis/virtualenv/python3.6.7/lib/python3.6/site-packages/sortedcollections/recipes.py", line 167, in __setitem__

    self._dict_delitem(key)

AttributeError: 'ValueSortedDict' object has no attribute '_dict_delitem'
``

**Comment by grantjenks** [comment]
That's more a bug in sortedcollections than in sortedcontainers.

It's another good argument for moving ValueSortedDict into sortedcontainers #57 .

The issue has been fixed in version 1.2.1 of sortedcollections on PyPI.

**Comment by gbtami** [comment]
Thx a lot!
